dipy/tracking/tests/test_utils.py

```diff
@@ -353,11 +354,27 @@ def test_near_roi():
     npt.assert_array_equal(near_roi(streamlines, np.eye(4), mask),
                            np.array([False, True, False]))
 
+    # test for handling of various forms of null streamlines
+    # including a streamline from previous test because near_roi / tol
+    # can't handle completely empty streamline collections
+    streamlinesNULL = [np.array([[0., 0., 0.9],
+                                 [1.9, 0., 0.],
+                                 [3, 2., 2.]]),
+                       np.array([[],
+                                 [],
+                                 []]).T,
+                       np.array([]),
+                       []]
+    npt.assert_array_equal(near_roi(streamlinesNULL, np.eye(4), mask, tol=1),
+                           np.array([True, False, False, False]))
+    npt.assert_array_equal(near_roi(streamlinesNULL, np.eye(4), mask),
+                           np.array([False, False, False, False]))
+
     # If there is an affine, we need to use it:
     affine = np.eye(4)
     affine[:, 3] = [-1, 100, -20, 1]
     # Transform the streamlines:
-    x_streamlines = [sl + affine[:3, 3] for sl in streamlines]
+    x_streamlines = [sl + affine[:3, 3] for sl in streamlines[-3:]]
```

Comment:
Why is this needed? It doesn't seem to be referring to the new variable that you created. Is that what you meant to do here? I.e., 

```
 x_streamlines = [sl + affine[:3, 3] for sl in streamlinesNull[-3:]]
```

Reply:
I actually don't know what this is... this (to my understanding) doesn't really have anything to do with the changes I'd intended.  It might be a typo?  I believe it should be left in the previous state (i.e. no [-3:])